packages/client/webapp/src/utils/TA2/stratify.ts

```diff
@@ -0,0 +1,25 @@
+// Providing the ID of 3 Models (model A, model B, and the type Model)
+// Create a new model of based off of the stratification
+
+import { json } from 'd3';
+
+// TODO: Better error handling (models not found for example)
+//       Override function to also accept 2 typed models (need to chat with TA2 to fully understand)
+export async function stratify(modelA, modelB, typeModel) {
+	console.log('Start Stratify');
+
+	if (!modelA || !modelB || !typeModel) {
+		return json('An ID must be provided for each model');
+	}
+	console.log(`Stratifying: ${modelA} ${modelB} ${typeModel}`);
```

Comment:
I think it's a good habit to get into to clear out these console.log()s before raising your PR, it'll slow down debugging in the future if we end up needing to dig through the logs from when each feature was being developed.